packages/edit-site/src/components/global-styles/typography-panel.js

```diff
@@ -54,8 +54,10 @@ function useHasLetterSpacingControl( name ) {
 	);
 }
 
-export default function TypographyPanel( { name } ) {
+export default function TypographyPanel( { name, element } ) {
```

Comment:
Hey, I'd like us to revisit this conversation https://github.com/WordPress/gutenberg/pull/35264#discussion_r723221063

What are your thoughts about passing the underlying functions `useSetting`, `useStyle`, and `getSupportedGlobalStylesPanels` the parameters `path` and `context`? The context would contain any extra metadata we need (source, block name, element, etc). I've recently introduced this change for the server-side API to be future-proof in regards to addressing different templates down the road https://github.com/WordPress/gutenberg/pull/36610 

Not a blocker for this PR, although I'm now more convinced we should do it for 5.9 than I was a few months ago when we first discussed it.

Reply:
yep, I fine with that change but this is an internal API for the moment, so there's no real urgency yet.